tests/unit_tests/dft/source/compute_tests.cpp

```diff
@@ -50,12 +50,18 @@ class ComputeTests_real_real_out_of_place
 
 #define INSTANTIATE_TEST(PRECISION, DOMAIN, PLACE, LAYOUT, STORAGE)                              \
     TEST_P(ComputeTests##_##LAYOUT##PLACE, DOMAIN##_##PRECISION##_##PLACE##_##LAYOUT##STORAGE) { \
-        auto test =                                                                              \
-            DFT_Test<oneapi::mkl::dft::precision::PRECISION, oneapi::mkl::dft::domain::DOMAIN>{  \
-                std::get<0>(GetParam()), std::get<1>(GetParam()).sizes,                          \
-                std::get<1>(GetParam()).batches                                                  \
-            };                                                                                   \
-        EXPECT_TRUEORSKIP(test.test_##PLACE##_##LAYOUT##STORAGE());                              \
+        try {                                                                                    \
+            auto test =                                                                          \
+                DFT_Test<oneapi::mkl::dft::precision::PRECISION,                                 \
+                         oneapi::mkl::dft::domain::DOMAIN>{ std::get<0>(GetParam()),             \
+                                                            std::get<1>(GetParam()).sizes,       \
+                                                            std::get<1>(GetParam()).batches };   \
+            EXPECT_TRUE(test.test_##PLACE##_##LAYOUT##STORAGE());                                \
```

Comment:
We are still returning `test_skipped` in the cases USM or `double` are not supported so this should still be `EXPECT_TRUEORSKIP`.

Reply:
That's true. I think I missed this because the tests were passing instead. Fixed in https://github.com/oneapi-src/oneMKL/pull/311/commits/0f0f28745a69f38dfa057fb159a45f98ef2ea6b3.